### PR TITLE
[MeshingApplication][MMG] Add option to specify mmg prefix

### PR DESCRIPTION
**Pull request by marcnunezc**
This allows to specify the prefix in the configure file. I had to add this to make use of the "MMG_ROOT" option and to avoid to add all the libraries individually.  Our configure looks like this in this case:

```cmd
rem Configure
@echo on
cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"          ^
-DUSE_EIGEN_MKL=OFF ^
-DINCLUDE_MMG=ON ^
-DMMG_ROOT="C:/Program Files (x86)/mmg" ^
-DMMG_LIBRARY_PREFIX=""

```

By default it is set to "lib". 

In Windows, MMG libraries are written without the "lib" prefix in my machine.

### CODE REVIEW

**Review by loumalouomega — APPROVED** [review]
Ok

File: applications/MeshingApplication/CMakeLists.txt [hunk 1]
```diff
@@ -58,22 +58,26 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 
 add_library(KratosMeshingCore SHARED ${KRATOS_MESHING_APPLICATION_CORE} ${KRATOS_MESHING_APPLICATION_CORE_MMG} ${KRATOS_MESHING_APPLICATION_TESTING_SOURCES})
 
+IF(NOT DEFINED MMG_LIBRARY_PREFIX)
+    SET(MMG_LIBRARY_PREFIX "lib")
+ENDIF(NOT DEFINED MMG_LIBRARY_PREFIX)
```
Comment (philbucher) [inline applications/MeshingApplication/CMakeLists.txt:63]: can you try this please @marcnunezc?
```suggestion
OPTION (MMG_LIBRARY_PREFIX "Prefix for MMG lib" "lib")
```

Comment (marcnunezc) [reply applications/MeshingApplication/CMakeLists.txt:63]: Ìf I dont set the MMG_LIBRARY_PREFIX in our configure, this returns:
MMG_LIBRARY_PREFIX="OFF"
which tries to look for the library name
"lib/OFFmmg.a"

Is this the expected behaviour?

Comment (philbucher) [reply applications/MeshingApplication/CMakeLists.txt:63]: What I thought of doesn't with string options, please neglect my comment, sorry for the noise and thx for testing :)

Comment (marcnunezc) [reply applications/MeshingApplication/CMakeLists.txt:63]: If it worked it would be much cleaner!

Comment (philbucher) [reply applications/MeshingApplication/CMakeLists.txt:63]: works only with ON/OFF unfortunately :/

**Comment by philbucher** [comment]
guess you need to merge master to start the CI

**Pull request merged**